nodejs/templates/_helpers.tpl

```diff
@@ -9,4 +9,55 @@ We truncate at 63 chars because some Kubernetes name fields are limited to this
 {{- else -}}
 {{- printf "%s-%s" .Release.Name .Chart.Name | trunc 63 | trimSuffix "-" -}}
 {{- end -}}
-{{- end -}}
\ No newline at end of file
+{{- end -}}
+
+---
+{{/*
+All the common lables needed for the lables sections of the definitions.
+*/}}
+{{- define "labels" }}
+app.kubernetes.io/name: {{ template "hmcts.releaseName" . }}
+helm.sh/chart: {{ .Chart.Name }}-{{ .Chart.Version }}
+app.kubernetes.io/managed-by: {{ .Release.Service }}
+app.kubernetes.io/instance: {{ template "hmcts.releaseName" . }}
+{{- end -}}
+
+{{/*
+The bit of templating needed to create the flex-Volume keyvault for mounting
+*/}}
+{{- define "secretVolumes" }}
+{{- if .Values.keyVaults }}
+{{- $globals := .Values.global }}
+{{- $keyVaults := .Values.keyVaults }}
+volumes:
+{{- range $vault, $info := .Values.keyVaults }}
+- name: {{ $vault }}
+  flexVolume:
+    driver: "azure/kv"
+    secretRef:
```

Comment:
i missed this in @luigibk 's PR but this should be optional and depending on whether another flag is set,
`usePodIdentity`

That will enable easier testing and swapping over,
We can remove the flag once we've migrated fully over

Reply:
oh ... you mean like feature flag this ?